src/App.tsx

```diff
@@ -1,17 +1,140 @@
-import React from 'react';
-import './App.css';
-
+import React, { useState } from 'react';
+import './App.scss';
 import users from './api/users';
+import todos from './api/todos';
+import TodoList from './components/TodoList/TodoList';
+
+const preparedUsers = users.map(user => {
+  return {
+    id: user.id,
+    name: user.name,
+    email: user.email,
+  };
+});
+
+const preparedTodos = todos.map(todo => {
+  return {
+    userId: todo.userId,
+    id: todo.id,
+    title: todo.title,
+    completed: todo.completed,
+    user: preparedUsers.find(user => user.id === todo.userId),
+  };
+});
 
 const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [userName, setUserName] = useState('');
+  const [todosList, setTodosList] = useState(preparedTodos);
+  const [titleError, setTitleError] = useState('');
+  const [userNameError, setUserNameError] = useState('');
+
+  const handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
```suggestion
  const handleTitleChange = (event: ChangeEvent<HTMLInputElement>) => {
```
`import { ChangeEvent } from 'react'` чтобы сократить запись